packages/captp/test/worker.js

```diff
@@ -1,10 +1,13 @@
 import '@endo/init/pre-remoting.js';
 import '@endo/init/debug.js';
 
-import { parentPort } from 'worker_threads';
+import { parentPort as maybeParentPort } from 'worker_threads';
 import { Fail } from '@endo/errors';
 import { makeGuest, makeHost } from './traplib.js';
 
+if (!maybeParentPort) throw new Error('null parentPort');
```

Comment:
I guess but do not know that this is safe in the face of port `0` as opposed to `undefined`. `0` is a special port for `listen`, but `connect` won’t ever provide an address with a port of `0` unless there’s an error.